it/controllers/propertyLinking/ClaimPropertyOwnershipISpec.scala

```diff
@@ -0,0 +1,1493 @@
+package controllers.propertyLinking
+
+import base.{HtmlComponentHelpers, ISpecBase}
+import binders.propertylinks.ClaimPropertyReturnToPage
+import com.github.tomakehurst.wiremock.client.WireMock._
+import com.github.tomakehurst.wiremock.stubbing.StubMapping
+import models.{ClientDetails, LinkingSession, Occupier, Owner, OwnerOccupier, PropertyOccupancy, PropertyOwnership, PropertyRelationship}
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import play.api.http.HeaderNames
+import play.api.http.Status.OK
+import play.api.libs.json.Json
+import play.api.libs.ws.WSResponse
+import play.api.test.Helpers._
+import repositories.PropertyLinkingSessionRepository
+import utils.ListYearsHelpers
+
+import java.time.LocalDate
+
+class ClaimPropertyOwnershipISpec extends ISpecBase with HtmlComponentHelpers with ListYearsHelpers {
+
+  lazy val mockPropertyLinkingSessionRepository: PropertyLinkingSessionRepository = app.injector.instanceOf[PropertyLinkingSessionRepository]
+  val localCouncilReferenceValue = "2050466366770"
+  val propertyAddress = "Test Address, Test Lane, T35 T3R"
+  val dayOfOwnership: Int = 1
+  val monthOfOwnership: Int = 1
+  val yearOfOwnership: Int = 2020
+  val occupancyEndDate: Option[PropertyOccupancy] = Some(PropertyOccupancy(stillOccupied = false, lastOccupiedDate = Some(LocalDate.of(2021, 1, 1))))
+
+  val titleText = "When you became the owner or occupier of the property - Valuation Office Agency - GOV.UK"
+  val titleTextAgent = "When your client became the owner or occupier of the property - Valuation Office Agency - GOV.UK"
+  val backLinkText = "Back"
+  val localCouncilText = s"Local council reference: $localCouncilReferenceValue"
+  val propertyDetailsText = s"Property: $propertyAddress"
+  val captionText = "Add a property"
+  val headingText = "When you became the owner or occupier of the property"
+  val headingTextAgent = "When your client became the owner or occupier of the property"
+  val iHaveOwnedText = "I have owned or occupied the property on more than one occasion"
+  val iHaveOwnedTextAgent = "My client has owned or occupied the property on more than one occasion"
+  val youCanAddText = "You can add the property for each period you were connected to it, but you need to do it one period at a time. However, if you want to talk to the Valuation Office Agency about the valuation, you will need to select the right valuation period."
+  val youCanAddTextAgent = "You can add the property for each period your client was connected to it, but you need to do it one period at a time. However, if you want to talk to the Valuation Office Agency about the valuation, you will need to select the right valuation period."
+  val onWhatDateText = "On what date did you become the owner or occupier?"
+  val onWhatDateTextAgent = "On what date did your client become the owner or occupier?"
+  val forExampleText = "For example, 1 4 2017"
```

Comment:
I think this forExampleText changes depending on if the property is English or Welsh. So for English its 1 4 2017 and for Welsh it's 1 4 2023 - It might be worth adding a test for that

Reply:
Isnt it set by whatever earliestStartDate is set in session, so even though English/Welsh is different, it'll only ever display whats there i.e. i could manually change it to 1 1 2010 and it'll display that

Reply:
Ah okay cool 👍